pyndustric/constants.py

```diff
@@ -81,7 +81,6 @@
     "min": 2,
     "xor": 2,
     "atan2": 2,
-    "dst": 2,
```

Comment:
Did Mindustry remove this? Haven't played it in a long while.

Reply:
Ya, I went through the operators and I couldn't find it.

Reply:
`dst` was renamed to `len`